### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by ZackWard-student**
Fixed values getting divided out.

Actually assigned the initialized question to the questions array.

Please note that this assignment was late since I had a baby girl!

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -22,10 +22,7 @@ public static void Main(string[] args)
             Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
         }
 
-        public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
-        {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
-        }
+        public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions) => ((int)((float) numberCorrectAnswers / numberOfQuestions * 100)) + "%";
```
Comment (mmwoodfo) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:25]: I like the implementation of some class knowledge here. :)

Some comments... There is no need to cast this to an int, if you're looking to round your answer you should be more explicit about your intentions. Casting isn't bad, but it is often misused. To make your intentions clearer, use Math.Round or .ToString to preform rounding.

**Comment by mmwoodfo** [comment]
Congratulations on your baby girl :)

**Comment by mmwoodfo** [comment]
@ZackWard-student did you do a PR review? If so, can you provide me with the name of the student you did the review for and/or the PR #?

Thank you.

**Comment by ZackWard-student** [comment]
@mmwoodfo I did do a PR review, but I honestly didn't track who it was for. I can not find it. I did it as a comment on another students Pull request.

**Comment by mmwoodfo** [comment]
@ZackWard-student It's all good. I've found it. https://github.com/IntelliTect-Samples/EWU-CSCD371-2022-Winter/pull/42

Sorry, they can be difficult to track down sometimes. :)
I'll update your grade.

**Pull request closed without merging**